### PR TITLE
Adds upgrade doc for 2.16 to 2.17

### PR DESCRIPTION
**Pull request by imharshita**
Signed-off-by: Harshita <harshita.sharma6174@gmail.com>

Adds `Migrating to the Operator` upgrade document

### CODE REVIEW

File: content/kubermatic/v2.17/tutorials_howtos/upgrading/upgrade_from_2.16_to_2.17/chart_migration/_index.en.md [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+This chapter outlines a migration plan for the Kubermatic Operator, starting from having KKP installed
+using the old `kubermatic` Helm chart using Helm 2.x. or 3.x.
+
+{{% notice warning %}}
+Before performing the migration, make sure to back up all Kubermatic resources!
+{{% /notice %}}
+
+Make sure that you also have recent etcd snapshots for all user clusters.
+
+#### Overview
```
Comment (irozzo-1A) [inline content/kubermatic/v2.17/tutorials_howtos/upgrading/upgrade_from_2.16_to_2.17/chart_migration/_index.en.md:17]: I would put a warning mentioning that chart installations are not supported any longer at KKP 2.17 and that clusters have to be imperatively migrated. Wdyt?

**Comment by irozzo-1A** [comment]
LGTM I would just emphasize that migrating to operator is no longer an option ;-)

File: content/kubermatic/v2.17/tutorials_howtos/upgrading/upgrade_from_2.16_to_2.17/chart_migration/_index.en.md (hunk 1 above)
Comment (imharshita) [reply content/kubermatic/v2.17/tutorials_howtos/upgrading/upgrade_from_2.16_to_2.17/chart_migration/_index.en.md:17]: Sure I can put it.

Comment (imharshita) [reply content/kubermatic/v2.17/tutorials_howtos/upgrading/upgrade_from_2.16_to_2.17/chart_migration/_index.en.md:23]: done

**Pull request merged**